[VWROOM-276] Add some info about `dequeue_mode` (#1178)
Macrometacorp/docs: docs/photoniq/vwrs/configure-domain.md

- **queue_type**: Defines how requests should be removed from the waiting room. The three possible queue types are `fifo`, `random`, and `lottery`. If this is not set, then the default queue is `fifo`.
- **queue_mode**: You can configure the waiting room to be enabled dynamically. When set to `auto`, the waiting room is enabled after reaching the defined `rate_limit` for a specific `metric_interval`. If set to `manual`, then the waiting room is always enabled.

> - **dequeue_mode**: When the value is `on`, users in the waiting room will be granted access to the origin at a rate given by `rate_limit`. When the value is `off`, users will be kept in the waiting room indefinitely.

Reviewer: ```suggestion
- **dequeue_mode**: When the value is `on`, users in the waiting room are granted access to the origin at a rate given by `rate_limit`. When the value is `off`, users remain in the waiting room indefinitely.
```

> - **origin_key**: Every domain is associated with an origin that was created with the Create Origin REST API, POST `/api/vwr/v1/origins/{origin_id}`.

Reviewer 2: Did we not remove the origins api?  Should we remove this field?
